x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/google_workspace_admin_role_deletion.json

```diff
@@ -33,5 +33,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 10
+  "version": 12
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950